jina/serve/runtimes/gateway/http_fastapi_app_docarrayv2.py

```diff
@@ -252,35 +252,14 @@ def add_streaming_routes(
             methods=['GET'],
             summary=f'Streaming Endpoint {endpoint_path}',
         )
-        async def streaming_get(request: Request):
-            query_params = dict(request.query_params)
+        async def streaming_get(request: Request, body: input_doc_model = None):
+            if not body:
```

Comment:
We can add a conditionals like `and request.method == 'GET'` here.

Reply:
I think we may not need `POST` at all then.

Reply:
That's the current state of this PR, and it fixes everything. Can't tell if tests need to be re-run or if they're actually failing.

Reply:
you can see a test failing here https://github.com/jina-ai/jina/actions/runs/6647328275/job/18084538375?pr=6098

Reply:
I fixed that issue, I think the current failures just need re-runs, can you take a look?

Reply:
Nevermind, found one that's not flakyness